packages/devtools/cli/src/commands/halo/join.ts

```diff
@@ -50,6 +72,8 @@ export default class Join extends BaseCommand<typeof Join> {
           onConnecting: async () => ux.action.stop(),
           onReadyForAuth: async () => secret ?? ux.prompt(chalk`\n{red Secret}`),
           onSuccess: async () => {
+            ux.log('setting device profile...');
+            await updateDeviceProfile(client, label);
```

Comment:
Maybe make `client.halo.join` accept deviceProfile same as `client.halo.create`?

Reply:
yeah, I had considered that but was trying to avoid having to make such a big API change. but you are probably right, it'll ensure consistency in the browser and make it easier on the caller.